docs/Dockerfile

```diff
@@ -9,20 +9,19 @@ COPY  docs/references /app/src/app/docs/references
 COPY  docs/page.md /app/src/app/docs
 COPY  docs/navigation.js /app/src/lib
 
-
 ENV NODE_ENV production
 
 RUN npm install
 RUN npm run build
 
-RUN addgroup -g 1001 -S nodejs
-RUN adduser -S nextjs -u 1001
-
-USER nextjs
+# Stage 2: Serve with Static Server
+FROM zopdev/static-server:latest
```

Comment:
Using latest may have unintended consequences. We should fix the version instead.